README.md

```diff
@@ -9,6 +9,26 @@ Major features include:
 - Interactive console with auto-completion
 - Requires no special permissions
 
+## Getting Started
+Install the plugin using the Notepad++ Plugin Manager, or download it from the [Release](https://github.com/dail8859/LuaScript/releases) page and copy `LuaScript.dll` to your `\plugins` folder.
+
+#### Startup Scripts
+LuaScript looks for the file `\plugins\config\startup.lua` and automatically runs it on Notepad++ startup. You can include any commands you want to immediately execute on prgram startup, as well as register any additional functions/shortcuts.
```

Comment:
I understand, then perhaps a link from the readme to the documentation which describes varous methods to load/autoload scripts with their pros & cons. Or, even better you could include some code in the base plugin which does this cleanly for a specified folder.